digdag-standards/src/main/java/io/digdag/standards/operator/HttpOperatorFactory.java

```diff
@@ -254,6 +254,15 @@ private ContentResponse execute(Request req, boolean uriIsSecret)
             catch (HttpResponseException e) {
                 throw error(req, uriIsSecret, e.getResponse());
             }
+            catch (RuntimeException e) {
+                logger.info("Exception without response: {} {}", req.getMethod(), safeUri);
```

Comment:
This message should be output as `warn` not `info`

Reply:
Thanks for your comment. I fixed it.